packages/bpk-component-banner-alert/src/BpkBannerAlert.js

```diff
@@ -127,7 +163,7 @@ class BpkBannerAlert extends Component {
     */
     // Disabling 'click-events-have-key-events and interactive-supports-focus' because header element is not focusable.
     // ToggleButton is focusable and works for this.
-    return (
+    return show ? (
```

Comment:
this nested ternary operators are really hard to read, can we refactor the logic a bit?
1. if `!show` return null straightaway 
2.  create a const for the expandible and dismissable first set them to null if no needed
3. return the rendered component adding inside the const created before